Makefile

```diff
@@ -145,7 +145,7 @@ test-ci: build-standalone-ci
 
 test-ci-coverage:
 	BABEL_ENV=test BABEL_COVERAGE=true $(MAKE) bootstrap
-	BABEL_ENV=test BABEL_COVERAGE=true $(YARN) jest --maxWorkers=100% --ci
+	BABEL_ENV=test BABEL_COVERAGE=true $(YARN) jest --maxWorkers=100% --ci --coverage
```

Comment:
When `BABEL_COVERAGE` is true, we will build Babel with `babel-plugin-istanbul` and the coverage reporter should have worked already. I am not sure whether pulling `--coverage` will transform the instrumented code again.

Reply:
https://github.com/babel/babel/pull/14802/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R146-R148

This was a regression I introduced that resulted in our repo not uploading new coverage reports for a month. 🤦‍♂️

Reply:
Ok. Good to see the test coverage is 93% now.

Reply:
In fact, I have a local branch that improves `light-jest-runner` and adds sub-process coverage, but it seems to have encountered some problems with source map merging, which made it impossible to officially open PR.

Reply:
It's been a long time, maybe I should dig into the source map when I have time.🙂